paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettingDefinition.kt

```diff
@@ -157,6 +158,25 @@ internal interface PlaygroundSettingDefinition<T> {
         }
     }
 
+    data class LinkControllerConfigurationData(
+        private val configurationBuilder: LinkController.Configuration.Builder,
+        private val billingDetailsCollectionConfigurationBuilder: BillingDetailsCollectionConfigurationBuilder =
+            BillingDetailsCollectionConfigurationBuilder()
+    ) {
+        // Billing details is a nested configuration, but we have individual settings for it in the
+        // UI, this helper keeps all of the configurations, rather than just the most recent.
+        fun updateBillingDetails(
+            block: BillingDetailsCollectionConfigurationBuilder.() -> Unit
+        ) {
+            billingDetailsCollectionConfigurationBuilder.apply {
+                block()
+            }
+            configurationBuilder.billingDetailsCollectionConfiguration(
+                billingDetailsCollectionConfigurationBuilder.build()
+            )
+        }
+    }
+
```

Comment:
I'm liking the idea of moving LinkController out of the `paymentsheet` playground to its own activity to avoid mixing configurations.